apps/system/js/app_window.js

```diff
@@ -1286,7 +1286,7 @@
         this.frontWindow._showScreenshotOverlay();
         return;
       }
-      if (!this.screenshotOverlay ||
+      if (!this.element || !this.screenshotOverlay ||
```

Comment:
Will this make a difference? Looks like `this.screenshotOverlay` was statically assigned from a selector query in line 672, rather than being a dynamic access. Or do we have magic so the query is lazy?